code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -210,7 +210,7 @@
 	name = "Disguise"
 	mechanics_text = "Disguise yourself as the enemy. Uses plasma to move."
 	///the regular appearance of the hunter
-	var/old_appearance
+	var/atom/old_appearance
```

Comment:
i said appearance, it's not an atom

http://www.byond.com/docs/ref/index.html#/atom/var/appearance

Reply:
The reason why I'm defining the type of old_appearance is to access the layer property of the appearance.

How do you suggest I access the layer of the variable without defining the type?

Reply:
Mutable_appearance?